hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -177,21 +185,21 @@ struct InputVarInfo {
 struct FCInputInfo {
     struct InputInfo a, b, c, d, e, f, g;
     int c0, c1;
-    //uint32_t c0_value, c1_value;
+    // uint32_t c0_value, c1_value;
```

Comment:
Probably doesn't belong in this PR, but we should probably remove this, or add a comment instead as to why these are not here (I actually don't know from the top of my head).